config/samples/flows_v1alpha1_flowcollector_ebpf.yaml

```diff
@@ -8,6 +8,13 @@ spec:
   ebpf:
     image: 'quay.io/netobserv/netobserv-ebpf-agent:main'
     imagePullPolicy: IfNotPresent
+    sampling: 0
+    cacheActiveTimeout: 5s
+    cacheMaxFlows: 1000
+    interfaces: []
+    excludeInterfaces: ["lo"]
+    buffersLength: 50
+    verbose: true
```

Comment:
not necessarily for this PR (as it would also impact the agent), but can we change that to `logLevel` like we're doing for the other components, or is there a reason why it needs to be a bool ?

Reply:
It's fine. I'll address the observation into this PR. Thanks!

Reply:
Do'h! I approved before submitting the previous PR. The verbose-to-loglevel change has been added here: https://github.com/netobserv/network-observability-operator/pull/82